### DISCUSSION TITLE
Authorization for Attu

### DISCUSSION THREAD

**Discussion by deshetti** [milvus-io/milvus#17536] [Q&A and General discussion] [answered]
Is there a plan to add authorization for Attu? Currently, if we install Milvus & Attu with a Helm chart, we can connect to the Milvus instance directly and perform operations on the database which is insecure. I hear that authorization is planned in the upcoming version of Milvus, do you plan to add the same for Attu as well?

**Comment by xiaofan-luan** [comment, marked as answer]
We have Role based access control in some release of 2.1, then attu will be modified to support all authorization

**Reply by deshetti** [reply]
Thanks for the quick response.

**Comment by shanghaikid** [comment]
alone with the next release of milvus, attu will support this feature.
